### PR TITLE
feat(standards): add superclassRole to ariaRoles

### PR DESCRIPTION
**Pull request by redgardner**
Added superclassRole attributes as an array of strings per WAI-ARIA 1.1 spec

Closes issue #2151

Added superclassRole to each aria-role per WAI-ARIA 1.1 spec

Closes issue:

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Follows the commit message policy, appropriate for next version
- [x] Code is reviewed for security

### CODE REVIEW

**Comment by straker** [comment]
Thanks for the pr. Would you be able to add the `superclassRole` to the ARIA 1.2 roles as well? For example [insertion](https://www.w3.org/TR/wai-aria-1.2/#insertion)

**Comment by redgardner** [comment]
Sure thing! I'll make this update

**Comment by redgardner** [comment]
Made the update. Is it correct to wait to squash these commits until merge to develop?

**Comment by straker** [comment]
Yep. We'll squash the commit on our side so you don't have to worry about it.

File: lib/standards/aria-roles.js
Comment (WilcoFiers) [inline lib/standards/aria-roles.js:668]:
```suggestion
		superclassRole: ['section'],
		// Note: spec difference
```

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
Couple minor edits. Should be good to go after that.

File: lib/standards/aria-roles.js
Comment (redgardner) [reply lib/standards/aria-roles.js]: Resolved this with the latest push and a few more areas where I had done this

File: lib/standards/aria-roles.js
Comment (redgardner) [reply lib/standards/aria-roles.js]: Fixed

File: lib/standards/aria-roles.js
Comment (redgardner) [reply lib/standards/aria-roles.js]: Whoops. Thanks for the catch

File: lib/standards/aria-roles.js
Comment (redgardner) [reply lib/standards/aria-roles.js]: Fixed along with a few others I found

**Comment by redgardner** [comment]
Thanks I've addressed all the feedback and pushed up a new commit. I wasn't sure if that was the preferred way or accepting change recommendation.

**Review by WilcoFiers — APPROVED** [review]

**Comment by WilcoFiers** [comment]
@redgardner can you please click to sign the CLA. The link is https://cla-assistant.io/dequelabs/axe-core?pullRequest=2600

**Comment by redgardner** [comment]
@WilcoFiers I have tried to sign the CLA in multiple browsers and windows. I've gotten confirmation that I've signed it, but I keep getting the pending CLA note. 

I've tried hitting the recheck as well. Is there something else to try?

**Comment by WilcoFiers** [comment]
@redgardner I'm not sure what's going on with the CLA site. We've been having some issues. I'll take your message as confirmation you agree, and will merge the pull request. Thank you very much for this contribution!

**Comment by redgardner** [comment]
Thanks! Sorry for the hassle with the CLA stuff. I agree with the CLA.

**Pull request merged**
